pages/linux/mcookie.md

```diff
@@ -0,0 +1,19 @@
+# mcookie
+
+> Generate a 128-bit random hexadecimal number.
+
+- Generate random number:
+
+`mcookie`
+
+- Generate random number, use destination file as a seed for the randomness:
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a random number, using the contents of a file as a seed for the randomness:
```

Also, are there long forms of these options? In tldr-pages we prefer to use long forms of arguments to improve clarity.

Reply:
> In tldr-pages we prefer to use long forms of arguments to improve clarity.

Ok, I'll change it right away.